app/models/manageiq/providers/azure_stack/cloud_manager/metrics_capture.rb

```diff
@@ -51,7 +51,7 @@ def perf_collect_metrics(interval_name, start_time = nil, end_time = nil)
     start_time   = start_time.utc
 
     begin
-      target.ext_management_system.with_provider_connection do |connection|
+      target.ext_management_system.with_provider_connection do |_connection|
```

Comment:
@djberg96 lets just delete this whole file if it isn't implemented, this is the boilerplate provider generator version and doesn't do anything.

Reply:
Why do I have a feeling something will go horribly wrong if this file isn't present.

Reply:
Well there''s no `AzureStack::MetricsCaptureWorker` so as long as we remove the `require_nested :MetricsCapture` from `AzureStack::CloudManager` there aren't any references to `perf_collect_metrics`